tests/coreutils/du.rs

```diff
@@ -0,0 +1,154 @@
+use std::ffi::OsString;
+use uutils_args::{Arguments, Options};
+
+#[derive(Arguments)]
+enum Arg {
+    #[arg("--apparent-size")]
+    ApparentSize,
+
+    #[arg("-B[SIZE]")]
+    #[arg("--block-size[=SIZE]")]
```

Comment:
It should be possible to merge this, which (if I recall correctly) also makes a difference for the `--help`. It will print them as one argument instead of 2. Other than that, LGTM!

Reply:
Oh! I clearly remember making this *exact* change, so apparently I made a push mistake or something along those lines. Thanks, will fix!